synapseclient/models/file.py

```diff
@@ -68,8 +68,10 @@ class FileHandle:
         Synapse will generate this ID when the FileHandle is created."""
 
     etag: Optional[str] = None
-    """FileHandles are immutable from the perspective of the API. The only field that can
-        be change is the previewId. When a new previewId is set, the etag will change."""
+    """
+    FileHandles are immutable from the perspective of the API. The only field that can
+    be change is the previewId. When a new previewId is set, the etag will change.
```

Comment:
_"The only field that can be changed is the `previewId`"_

Are we sure this is correct? I thought `downloadAs` was part of the file handle.

Reply:
This was taken directly from:
https://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/file/FileHandle.html